vent/sqldb/adapters/pgsql_adapter.go

```diff
@@ -0,0 +1,355 @@
+package adapters
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/lib/pq"
+	"github.com/monax/bosmarmot/vent/logger"
+	"github.com/monax/bosmarmot/vent/types"
+)
+
+//SQLDB implements DBAdapter
+type SQLDB struct {
+	DB     *sql.DB
+	Log    *logger.Logger
+	Schema string
+	DBURL  string
+}
+
+// NewSQLDB connects to a SQL database and creates default schema and _bosmarmot_log if missing
+func NewSQLDB(dbURL string, schema string, l *logger.Logger) DBAdapter {
+	return &SQLDB{
+		Log:    l,
+		Schema: schema,
+		DBURL:  dbURL,
+	}
+}
+
+// Open connects to a SQL database and creates default schema and _bosmarmot_log if missing
+func (db *SQLDB) Open() error {
+	db.Log.Info("msg", "Connecting to database", "value", db.DBURL)
+
+	dbc, err := sql.Open("postgres", db.DBURL)
+	if err != nil {
+		db.Log.Error("msg", "Error opening database connection", "err", err)
+		return err
+	}
+	db.DB = dbc
+
+	if err = db.Ping(); err != nil {
+		db.Log.Error("msg", "Error database not available", "err", err)
+		return err
+	}
+
+	var found bool
+	found, err = db.findDefaultSchema()
+	if err != nil {
+		return err
+	}
+
+	if !found {
+		if err = db.createDefaultSchema(); err != nil {
+			return err
+		}
+	}
+
+	// create _bosmarmot_log
+	if err = db.SynchronizeDB(getLogTableDef()); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Close database connection
+func (db *SQLDB) Close() {
+	if err := db.DB.Close(); err != nil {
+		db.Log.Error("msg", "Error closing database", "err", err)
+	}
+}
+
+// Ping database
+func (db *SQLDB) Ping() error {
+	err := db.DB.Ping()
+	if err != nil {
+		db.Log.Debug("msg", "Error database not disponible", "err", err)
+	}
```

Comment:
`db.Log.Debug("msg", "Error database not disponible", "err", err)` should be
`db.Log.Debug("msg", "Error database not available", "err", err)`